src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Box, Container, Grid, Stack, Typography } from '@mui/material';
+import { AccordionItem, type AccordionItemProps } from './AccordionItem';
+export interface AccordionProps {
+  title: string;
+  subtitle?: string;
+  description?: string | JSX.Element;
+  accordionItems: AccordionItemProps[];
+  theme: 'light' | 'dark';
+
+  layout?: 'left' | 'center' | 'right';
+}
+
+export const Accordion: React.FC<AccordionProps> = ({
+  title,
+  subtitle,
+  description,
+  accordionItems,
+  theme,
+  layout = 'left',
+}) => {
+  const isDarkTheme = theme === 'dark';
+  const bgcolor = isDarkTheme ? 'primary.main' : '#FAFAFA';
+  const textColor = isDarkTheme ? 'white' : 'text.primary';
+
+  const isCenterLayout = layout === 'center';
+  const textAlignment = isCenterLayout ? 'center' : 'left';
+
+  return (
+    <Box sx={{ py: { xs: 4, md: 10 } }} bgcolor={bgcolor} component="section">
+      <Container>
+        <Grid spacing={{ xs: 5, md: isCenterLayout ? 7 : 16 }} container>
+          <Grid item xs={12} md={isCenterLayout ? 12 : 4}>
+            <Stack px={{ xs: 1, md: 0 }} spacing={{ xs: 2, md: 5 }}>
+              {/** Title */}
+              <Typography
+                textAlign={textAlignment}
+                color={textColor}
+                variant="h4"
+              >
+                {title}
+              </Typography>
+
+              {/** Subtitle */}
+              <Typography
+                textAlign={textAlignment}
+                color={textColor}
+                variant="h6"
+              >
+                {subtitle}
+              </Typography>
+
```

Comment:
@Carminepo2 subtitle is an optional prop. When it's undefined, an useless and empty h6 tag will be rendered.